### PR TITLE
how to use github

### PR DESCRIPTION
**Pull request by MadderRagax**
I hope I didn't break anything

### CODE REVIEW

**Comment by Naamloos** [comment]
Your version only seems to update the local copy of the ModManager. Could you make it update `GameData.Instance.Mods`?

**Comment by MadderRagax** [comment]
That shouldn't be the case because it's only a new reference to the same instance and updates to it should update `GameData.Instance.Mods` as well.

**Comment by Naamloos** [comment]
Uh, it should be

**Comment by Naamloos** [comment]
It's not a reference, your copying the static instance to a new var

**Comment by Naamloos** [comment]
you're**

File: Rizumu/GuiObjects/ModSelector.cs [hunk 1]
```diff
@@ -91,67 +75,54 @@ public void Draw()
...
+                case 7 when keyboard.IsKeyPress(Keys.Left) || keyboard.IsKeyPress(Keys.Right):
+                    mods.NoFail = !mods.NoFail;
+                    break;
+                case 8 when keyboard.IsKeyPress(Keys.Left) || keyboard.IsKeyPress(Keys.Right):
+                    mods.Instafail = !mods.Instafail;
+                    break;
+                case 9 when keyboard.IsKeyPress(Keys.Left) || keyboard.IsKeyPress(Keys.Right):
+                    mods.RotationMode = !mods.RotationMode;
+                    break;
             }
```
Comment (Naamloos) [inline Rizumu/GuiObjects/ModSelector.cs:126]: please add
`GameData.Instance.Mods = mods;`

**Pull request merged**